source/distributed/tria.cc

```diff
@@ -929,7 +929,7 @@ namespace
     // A lambda to avoid code duplication.
     const auto copy_vertex = [&](unsigned int vertex_index) -> void {
       // copy into local struct
-      for (size_t d = 0; d < dim; ++d)
+      for (int d = 0; d < dim; ++d)
```

Comment:
We use `unsigned int` in almost all of these places. Would you mind changing it? Same for the other place.

Reply:
Done!